px4_ros2_cpp/include/px4_ros2/control/setpoint_types/experimental/fixedwing_lateral_longitudinal.hpp

```diff
@@ -33,26 +36,50 @@ class FwLateralLongitudinalSetpointType : public SetpointBase
   Configuration getConfiguration() override;
 
   /**
-   * @brief Update
+   * @brief Update the setpoint with full flexibility by passing a FwLateralLongitudinalSetpointType
+   * and FwControlLimits struct.
+   *
+   * @param setpoint a FwLateralLongitudinalSetpoint object where course, airspeed
+   * direction, lateral acceleration, altitude, height rate and equivalent airspeed
+   * can be set for full flexibility. If multiple lateral inputs are set, the lower
+   * level inputs are used as feedforward. If a height rate is set, the altitude setpoint
```

Comment:
@mahima-yoga We actually document this quite clearly in the message definitions (px4_msgs). Should we refer to that here for more clarification?